front/app/modules/commercial/id_oostende_rrn/components/VerificationFormOostendeRrn.tsx

```diff
@@ -95,9 +89,6 @@ const VerificationFormOostendeRrn = memo<Props & WrappedComponentProps>(
             }
 
             queryClient.invalidateQueries({ queryKey: meKeys.all() });
-            await streams.fetchAllWith({
-              apiEndpoint: endpointsToRefetch,
-            });
```

Comment:
I couldn't find this endpoint being used anywhere in the codebase, actually. I think it might be an old one and was replaced by something else. Or am I wrong?

Reply:
Ah yes I think you're right :) Looks like an old one.